commit 786d6516877dc852392117d44a77bf3093c39dbd
Author: Pan Xiuli <xiuli.pan@linux.intel.com>
Date:   Wed Apr 15 15:28:01 2020 -0500

    ASoC: SOF: add probe support extend data
    
    Share how many injection probe DMAs and how many probe points
    driver can request from FW.
    injection_dmas_max 0 means injection is not supported
    probe_points_max 0 means whole probes subsystem in FW is not enabled
    and not compiled in.
    
    ABI version change to 3.14.0
    
    Signed-off-by: Pan Xiuli <xiuli.pan@linux.intel.com>
    Signed-off-by: Pierre-Louis Bossart <pierre-louis.bossart@linux.intel.com>
    Reviewed-by: Kai Vehmanen <kai.vehmanen@linux.intel.com>
    Reviewed-by: Guennadi Liakhovetski <guennadi.liakhovetski@linux.intel.com>
    Link: https://lore.kernel.org/r/20200415202816.934-10-pierre-louis.bossart@linux.intel.com
    Signed-off-by: Mark Brown <broonie@kernel.org>

diff --git a/include/uapi/sound/sof/abi.h b/include/uapi/sound/sof/abi.h
index 5995b79d6df1..e0fa2939d49c 100644
--- a/include/uapi/sound/sof/abi.h
+++ b/include/uapi/sound/sof/abi.h
@@ -26,7 +26,7 @@
 
 /* SOF ABI version major, minor and patch numbers */
 #define SOF_ABI_MAJOR 3
-#define SOF_ABI_MINOR 13
+#define SOF_ABI_MINOR 14
 #define SOF_ABI_PATCH 0
 
 /* SOF ABI version number. Format within 32bit word is MMmmmppp */